### ISSUE TITLE
Lots of warnings 'Found "extension <Type>" of type for which there is no original type declaration information' with v1.2.0

### ISSUE THREAD

**Issue by Lutzifer** [krzysztofzablocki/Sourcery#903]
Since Version 1.2.0 I have a lot of warnings like:

`Found "extension Dictionary" of type for which there is no original type declaration information`

This happens to Foundation classes (Dictionary, Collections, UnitDuration, Measurement, FileManager), classes from Pods (RxSwift.Observable) and classes defined in the current app.

Config file:

```
project:
  file: ../../${N42_PROJECT_NAME}.xcodeproj
  target:
    name: ${N42_PRODUCT_NAME}
templates:
  - SourceryTemplates/
output: ../../Sources/App/Release/Sourcery
args:
  additionalImports: |
    import ${N42_KIT_NAME}
```

Maybe related to https://github.com/krzysztofzablocki/Sourcery/issues/887 ?

**Comment by krzysztofzablocki** [comment]
@Lutzifer those kind of messages are expected, Over last couple of releases I've added more warnings during parsing so that the users of Sourcery know what kind of types aren't fully defined.

But if you are seeing this on types you've defined in the source code Sourcery scans let me know more details, I think there should only be messages related to either ambigious types or type inference on variable defintions

**Comment by Lutzifer** [comment]
Yeah, upon closer investigation, all types that had a warning where either in a pod, foundation or a local module.

As sourcery runs in the build phase in our case: What do you think about making this configurable to not yield a warning but info or something like that?

**Comment by krzysztofzablocki** [comment]
@Lutzifer is this turning into actual Xcode warnings for you? you can always use `--quiet` argument and that should suppress anything except errors

**Comment by Lutzifer** [comment]
yep, its Xcode warnings, because the format matches.

Doing
` sourcery --config sourcery.yml | sed "s/warning: Found \"extension/info: Found \"extension/g"`
now to change it before Xcode sees it.

**Comment by krzysztofzablocki** [comment]
right, I guess we should change it to info then, It's too verbose for warnings

**Comment by krzysztofzablocki** [comment]
Released 1.2.1 which changes them to info

**Comment by Lutzifer** [comment]
Cool, thanks for the fast release.
